cmd/minishift/cmd/start.go

```diff
@@ -127,6 +128,12 @@ func runStart(cmd *cobra.Command, args []string) {
 	setOcProxy()
 	setShellProxy()
 
+	if viper.IsSet("http-proxy") {
```

Comment:
So you are saying this communication is not using HTTPS? What are the URLS which are getting accessed?